File: test/test-assign/r-correct/student/tests/q3.R
test = list(
  name = "q3",
  cases = list(
    ottr::TestCase$new(
      hidden = FALSE,
      name = "q3a",
      points = ,
      code = {
        test_that("q3a", {
            expect_equal(nine, 9)
        })
      }
    ),
    ottr::TestCase$new(
      hidden = FALSE,
      name = "q3b",
      points = ,
      code = {
        test_that("q3b", {
            expect_equal(square(16), 256)
        })
      }
    )

  )
)

File: test/test_check/files/logs-tests/q3.py
OK_FORMAT = True

test = {
	"name": "q3",
	"points": 2,
	"hidden": False,
	"suites": [ 
		{
			"cases": [ 
				{
					"code": r"""
					>>> x
					True
					""",
					"hidden": False,
					"locked": False,
				}, 
				{
					"code": r"""
					>>> if x:
					... 	print("yep")
					... else:
					... 	print("nope")
					... 
					yep
					""",
					"hidden": True,
					"locked": False,
				}, 
			],
			"scored": False,
			"setup": "",
			"teardown": "",
			"type": "doctest"
		}
	]
}